app/helpers/follows_helper.rb

```diff
@@ -52,6 +52,18 @@ def unfollow_drop_id(followable)
     "unfollow-drop-#{entity_full_name(followable)}-#{followable.id}"
   end
 
+  def entity_title(title)
+    entity = title.gsub('::', '/').downcase
+    t("activerecord.models.#{entity}.other")
+  end
+
+  def entity_icon(entity)
+    case entity
```

Comment:
Have you considered the option to use a hash here instead of a case statement? Something like:

```ruby
def entity_icon(entity)
  {
    'Proposal' => 'proposals',
    'Budget::Investment' => 'budget'
  }[entity]
end
```

It could be improved by moving the hash to a constant variable, or even switching keys for values to use new hash syntax:
```
def entity_icon(entity)
  {
    proposals: 'Proposal',
    budget: 'Budget::Investment'
  }.invert[entity]
end
```

Last but not least, it seems odd that "proposals" is plural bug "budget" is singular 🤔